commands/command_utils.py

```diff
@@ -0,0 +1,150 @@
+import json
+import os
+import re
+
+from leapp.exceptions import CommandError
+from leapp.utils import path
+
+HANA_BASE_PATH = '/hana/shared'
+HANA_SAPCONTROL_PATH = 'exe/linuxx86_64/hdb/sapcontrol'
+
+LEAPP_UPGRADE_FLAVOUR_DEFAULT = 'default'
+LEAPP_UPGRADE_FLAVOUR_SAP_HANA = 'saphana'
+LEAPP_UPGRADE_PATHS = 'upgrade_paths.json'
+
+
+# NOTE(ivasilev) Borrowed from https://semver.org/
+SEMVER_REGEX = re.compile((r"^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)"
+                           r"(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)"
+                           r"(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?"
+                           r"(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$"))
+
+
+def check_semver(version):
```

Comment:
@fernflower my question is, why? As you know, that only X.Y are expected, why to do something like this? there is no possible to find RHEL 8.yada or 8.7.3, as far as I know. So what is the reason behind such canon?

Reply:
Well it was you who wanted version checking, no? =)
Or did you mean "check that X and Y are numbers and be done with it"?

Reply:
Should be fixed now.